src/components/ContactDetails/ContactDetailsAvatar.vue

```diff
@@ -285,7 +310,6 @@ export default {
 		 */
 		removePhoto() {
 			this.contact.vCard.removeProperty('photo')
-			this.maximizeAvatar = !this.maximizeAvatar
```

Comment:
This allowed closing the modal on remove as you can also remove the avatar when displaying it full view

Reply:
I merged the two menus (that means no modal menu any more)

Reply:
Well, I would keep the modal menu anyway, no?

Reply:
I'd say we don't need the menu twice.

Reply:
But you cannot access it when the image is opened? SO it is extra clicks, as you need to close and then open the menu again?

Reply:
But I'll let  @jancborchardt decide :)

Reply:
It’s not bad to have the menu in both places. Sometimes you need to look at it in detail to decide. :)

Reply:
In order to prepare that, I moved the menu into a component. @skjnldsv maybe an intermediate review is possible before I start to include the menu also in the modal?

Reply:
Hey! You can't do that, the Actions component is handled by the Modal one.
That's why I said you'd have to duplicate code for now and put this into `<template #actions>`

Reply:
I thought with programming everything is possible... but ok, if I really have to, I will then just duplicate the code.